aiplatform/src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java

```diff
@@ -52,13 +53,16 @@ static void predictTextSentimentAnalysis(String project, String content, String
     try (PredictionServiceClient predictionServiceClient =
         PredictionServiceClient.create(predictionServiceSettings)) {
       String location = "us-central1";
-      String jsonString = "{\"content\": \"" + content + "\"}";
+
+      // Use JsonObject to ensure safe serialization of the input content; handles characters like `"`.
```

Comment:
Please fix this lint issue:

```
[ERROR] src/main/java/aiplatform/PredictTextSentimentAnalysisSample.java:[57] (sizes) LineLength: Line is longer than 100 characters (found 103).
```

Reply:
_eek_ sorry. fixed here https://github.com/GoogleCloudPlatform/java-docs-samples/commit/52d3f0929384e9cb6eb5ee3f6f3273edfeaff221